packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -55,7 +59,16 @@ const Layout = ({ layoutData }: LayoutProps): JSX.Element => {
           <div id="content">{route && <Placeholder name="headless-main" rendering={route} />}</div>
```

Comment:
We discussed, but just a reminder to cover the other 2 placeholders too ("headless-header" and "headless-main").